releases/release-1.31/release-notes/maps/pr-125279-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 125279
+releasenote:
+  text: |-
+    kube-scheduler implements scheduling hints for the VolumeRestriction plugin.
```

This wasn't resolved. 
```suggestion
    kube-scheduler implemented scheduling hints for the VolumeRestriction plugin.
```